src/main/java/AlexLink/HomeWork/HW7/Gradebook/Runner.java

```diff
@@ -0,0 +1,22 @@
+package AlexLink.HomeWork.HW7.Gradebook;
+
+//4. Создать класс Зачетная Книжка с внутренним классом, с помощью объек-
+//тов которого можно хранить информацию о сессиях, зачетах, экзаменах.
+
+import java.util.ArrayList;
+
+public class Runner {
+    public static void main(String[] args) {
+        ArrayList<GradeBook.Exam> examArrayList = new ArrayList<>();
```

Comment:
List instead of ArrayList. Work on interface level

Reply:
fixed it